blueprints/ember-cli-mirage/files/__root__/config.js

```diff
@@ -1,4 +1,22 @@
-export default function() {
+import {
+  discoverEmberDataModels,
+  // applyEmberDataSerializers,
+} from 'ember-cli-mirage';
+
+export default function (config) {
+  let finalConfig = {
+    ...config,
+    // Remove discoverEmberDataModels if you do not want ember-cli-mirage to auto discover the ember models
+    models: { ...discoverEmberDataModels(), ...config.models },
+    // uncomment to opt into ember-cli-mirage to auto discover ember serializers
+    // serializers: applyEmberDataSerializers(config.serializers),
+    routes,
+  };
+
+  return createServer(finalConfig);
```

Comment:
I was trying to update my app and use this alpha version, and found one missing detail here. 
In upgrade guide and this file missing import statement for `createServer`

Reply:
https://github.com/miragejs/ember-cli-mirage/pull/2457 to fix that. Thanks for pointing that out.